uisrnn/utils.py

```diff
@@ -310,3 +300,25 @@ def output_result(model_args, training_args, test_record):
   with open(filename, 'a') as file_object:
     file_object.write(output_string)
   return output_string
+
+
+def estimate_transition_bias(cluster_ids, smooth=1):
+  """estimate transition bias.
+
+  Args:
+    cluster_id: A list of cluster indicator sequences, must be a
+      non-concatenated sequence.
```

Comment:
Instead of say "must be", what about this:
cluster_id: Either a list of cluster indicator sequences, or a single concatenated sequence. The former is strongly preferred, since the transition_bias estimated from the latter will be inaccurate.

Reply:
OK, sure!